tests/create_test_niftis.py

```diff
@@ -0,0 +1,265 @@
+# Creates the various test NIfTI files used in the test suite from the source
```

Comment:
When does this script run?

Reply:
It's now integrated into the `pytest_configure` function which runs as setup before the tests.